cli/src/cli_util.rs

```diff
@@ -3088,56 +3087,60 @@ pub fn compute_commit_location(
                 Ok(None)
             }
         };
-    let destination_commits = resolve_revisions(destination)?;
-    let after_commits = resolve_revisions(insert_after)?;
-    let before_commits = resolve_revisions(insert_before)?;
-
-    let (new_parents, new_children) = match (destination_commits, after_commits, before_commits) {
-        (Some(destination_commits), None, None) => (destination_commits, vec![]),
-        (None, Some(after_commits), Some(before_commits)) => (after_commits, before_commits),
-        (None, Some(after_commits), None) => {
-            let new_children: Vec<_> =
-                RevsetExpression::commits(after_commits.iter().ids().cloned().collect_vec())
-                    .children()
-                    .evaluate(workspace_command.repo().as_ref())?
+    let destination_commit_ids = resolve_revisions(destination)?;
+    let after_commit_ids = resolve_revisions(insert_after)?;
+    let before_commit_ids = resolve_revisions(insert_before)?;
+
+    let (new_parent_ids, new_child_ids) =
+        match (destination_commit_ids, after_commit_ids, before_commit_ids) {
+            (Some(destination_commit_ids), None, None) => (destination_commit_ids, vec![]),
+            (None, Some(after_commit_ids), Some(before_commit_ids)) => {
+                (after_commit_ids, before_commit_ids)
+            }
+            (None, Some(after_commit_ids), None) => {
+                let new_child_ids: Vec<_> =
+                    RevsetExpression::commits(after_commit_ids.iter().cloned().collect_vec())
```

Comment:
nit: `after_commit_ids.clone()`

fwiw, I don't mind if all `Vec<CommitId>` variables are called `commits` here because this function works primarily on `CommitId`s.